test/functional/admin/republishing_controller_test.rb

```diff
@@ -19,6 +19,23 @@ class Admin::RepublishingControllerTest < ActionController::TestCase
     assert_response :forbidden
   end
 
+  test "GDS Admin users should be able to access GET :confirm with an existing document slug" do
+    get :confirm, params: { document_slug: "past-prime-ministers" }
+    assert_response :ok
+  end
+
+  test "GDS Admin users should see a 404 page when trying to confirm republishing a document that doesn't exist" do
+    get :confirm, params: { document_slug: "non-existent" }
+    assert_response :not_found
+  end
```

Comment:
And:
- `that doesn't exist` -> `that is not republishable via this interface`
- `document_slug: "not-republishable"`